src/mixins/object.svg_export.js

```diff
@@ -60,7 +60,6 @@
 
     /**
      * Returns styles-string for svg-export
-     * @param {Boolean} skipShadow a boolean to skip shadow filter output
```

Comment:
would you mind correcting the JSDOC here?
* @param {Object} style object to getthe style-string from